app/src/test/java/com/sdp/movemeet/Backend/Firebase/Firestore/FirestoreActivityManagerTest.java

```diff
@@ -374,12 +374,12 @@ public void addThrowsIllegalArgumentExceptionOnEmptyPath() {
         });
     }
 
-    @Test
-    public void addReturnsCorrectTask() {
-        Task<Void> result = fam.add(fakeActivity, "doc");
-        assertEquals(fakeActivity.getDocumentPath(), "doc");
-        assertEquals(fakeAddTask, result);
-    }
+//    @Test
```

Comment:
I'll make sure to adapt this test to both of our uses of `documentPath`

Reply:
Yes, that's kind of you, thank you!